Helm Chart updates and AWS session updates

Luke-Humphreys opened the pull request:
Adding a templated helm chart to deploy helm-chart-publisher to kubernetes. Also added more documentation on how to run within minikube.

Commit:
added support for aws sessions, replaced glide with dep, cleaned up

Luke-Humphreys committed:
Merge pull request #1 from HotelsDotCom/support-for-aws-sessions

added support for aws sessions, replaced glide with dep, cleaned up

Commit:
Untemplated helm chart

Fixed helm chart description

Refactored helm charts to use templating

Updated readme with minikube documentation

Updated gitignore

Added ingress

Added fixed image version

further chart templating

Commit:
Untemplated helm chart

Commit:
Fixed helm chart description

Commit:
Refactored helm charts to use templating

Commit:
Updated readme with minikube documentation

Commit:
Updated gitignore

Commit:
Added ingress

Commit:
Added fixed image version

Commit:
new line

Commit:
README update regarding mounting your AWS credentials in minikube

Commit:
merge master

Commit:
Pulled in changes, fixed conflicts

Luke-Humphreys committed:
Merge pull request #2 from HotelsDotCom/adding-helm-chart

Helm chart for helm-chart-publisher

Commit:
Updated gitignore

Luke-Humphreys committed:
Merge branch 'master' into merge-upstream-master

Luke-Humphreys committed:
Merge pull request #3 from HotelsDotCom/merge-upstream-master

Tested and works as expected, merge upstream master

Luke-Humphreys committed:
Merge branch 'master' into master

Commit:
Added templating for annotations

Commit:
Removed deployment from annotation

Luke-Humphreys committed:
Merge pull request #4 from HotelsDotCom/updating-helm-chart

Added templating for annotations

luizbafilho commented:
Thanks guys, let me know when you finish this up, so I can take I look and merge it.

Commit:
Added s3 protocol support

Commit:
Updated helm chart templates for s3 protocol

Commit:
Fixed indentation of values.yaml

Commit:
Bumped chart version

Commit:
Refactored new session

Commit:
Changed newSession to newS3Session

Luke-Humphreys committed:
Merge pull request #6 from HotelsDotCom/s3-protocol-support

S3 protocol support

Luke-Humphreys commented:
PR has got slightly larger, the main changes are:

- Added templating to the helm chart
- Updated documentation
- Added in an example values.yaml for helm chart
- Improved AWS sessions so tokens don’t time out
- Added support for s3 protocol instead of just http

Luke-Humphreys commented:
All good to take a look now :)

Commit:
changed dependency paths

Commit:
Merge pull request #7 from HotelsDotCom/change-import-paths

changed dependency paths
